src/worker.cc

```diff
@@ -85,7 +85,7 @@ const std::unordered_map<unsigned, valhalla::valhalla_exception_t> error_codes{
     {151, {151, "Exceeded max time", 400, HTTP_400, OSRM_INVALID_VALUE, "too_large_time"}},
     {152, {152, "Exceeded max contours", 400, HTTP_400, OSRM_INVALID_VALUE, "too_many_contours"}},
     {153, {153, "Too many shape points", 400, HTTP_400, OSRM_INVALID_VALUE, "too_large_shape"}},
-    {154, {154, "Path distance exceeds the max distance limit", 400, HTTP_400, OSRM_DISTANCE_EXCEEDED, "too_large_distance"}},
+    {154, {154, "Path distance exceeds the max distance limit, max distance is " + to_string(max_distance) + " Kilometers.", 400, HTTP_400, OSRM_DISTANCE_EXCEEDED, "too_large_distance"}},
```

Comment:
That can't work, `max_distance` is not defined anywhere for this block of code. `valhalla_exception_t` accepts an `extra` as a string, so every time `154` is raised, we'd need to pass the current `max_distance` within that extra string to the exception.

Reply:
valhalla_exception_t(unsigned code, const std::string& extra = "");
So by passing in the max_distance and converting to a string anywhere 154 is raised, this should pass the current max_distance as the variable "extra" so that it can be used in the error message?